tango_simlib/utilities/fandango_json_parser.py

```diff
@@ -58,6 +60,37 @@ def parse(self, json_file):
             elif data_component == "dev_class":
                 self.device_class_name = elements
 
+    def _flatten_device_attribute_dictionary(self):
+        for attribute_config in self._device_attributes.values():
+            _events_properties = attribute_config["events"]
+            events_properties = self._extract_attribute_events_properties(
+                _events_properties
+            )
+            attribute_config.update(events_properties)
+
+            alarms_properties = attribute_config["alarms"]
+            attribute_config.update(alarms_properties)
+
+    def _remove_unused_device_attributes_keys(self):
+        keys_to_pop = [
+            "alarms",
+            "color",
+            "database",
+            "device",
+            "dim_x",
+            "dim_y",
+            "events",
+            "extensions",
+            "format",
+            "model",
+            "string",
+            "time",
+        ]
+        for attribute_config in self._device_attributes.values():
+            # pop out keys not required to configure an attribute
```

Comment:
This comment seems unnecessary.  The method name already indicates that we are doing this.